<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyCert</class>
 <widget class="QWidget" name="VerifyCert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>264</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>305</width>
    <height>0</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">#name {
color: #003168;
font-family: Roboto, Helvetica;
font-size: 18px;
font-weight: 700;
}
#info, #validUntil {
color: #091A36;
font-family: Roboto, Helvetica;
font-size: 12px;
padding: 2px 10px;
border-radius: 8px;
background-color: #F3F5F7;
}
#info[label=&quot;error&quot;], #validUntil[label=&quot;error&quot;] {
color: #AD2A45;
background-color: #F5EBED;
}
#info[label=&quot;warning&quot;], #validUntil[label=&quot;warning&quot;] {
color: #83612D;
background-color: #FDF7EE;
}
#info[label=&quot;good&quot;], #validUntil[label=&quot;good&quot;] {
color: #1A641B;
background-color: #EAF8EA;
}
#changePIN {
color: #2F70B6;
border: 1px solid #2F70B6;
border-radius: 4px;
padding: 11px 12px;
font-family: Roboto, Helvetica;
font-size: 14px;
font-weight: 700;
}
#changePIN:default {
color: #ffffff;
border-color: #2F70B6;
background-color: #2F70B6;
}
#changePIN:default:hover {
border-color: #2B66A6;
background-color: #2B66A6;
}
#changePIN:default:pressed {
border-color: #215081;
background-color: #215081;
}</string>
  </property>
  <layout class="QVBoxLayout" name="VerifyCertLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="nameLayout" stretch="1,0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="nameLeft" native="true">
       <layout class="QHBoxLayout" name="nameRightLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="nameRightSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSvgWidget" name="nameIcon" native="true">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string notr="true">name</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="nameRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="Label" name="validUntil">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">Certificate is valid until %1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="buttonSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Label" name="info">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">PIN%1 has been blocked because PIN%1 code has been entered incorrectly 3 times.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QPushButton" name="changePIN">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string notr="true">Change PUK</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="buttonSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="links" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>57</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>57</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
color: #2F70B6;
border: none;
font-family: Roboto, Helvetica;
font-size: 14px;
text-decoration: underline solid;
}
QToolButton:hover {
background-color: #EAF1F8;
}
QToolButton:pressed {
background-color: #BFD3E8;
}</string>
     </property>
     <layout class="QVBoxLayout" name="widgetLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QToolButton" name="forgotPinLink">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string notr="true">Change with PUK code</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/icon_Abi.svg</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QToolButton" name="details">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Check the details of the certificate</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/icon_Abi.svg</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QToolButton" name="checkCert">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Check certificate status</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/icon_Abi.svg</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtSvg/QSvgWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>widgets/Label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
